File: 프로그래머스/0/181878. 원하는 문자열 찾기/원하는 문자열 찾기.cpp
#include <string>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;

int solution(string myString, string pat) {
    int answer = 0;
    
    string newMyString = "";
    string newPat = "";

    for(auto& i: myString){
        i=tolower(i);
    }
    for(auto& i: pat){
        i=tolower(i);
    }
    
    
    
    cout << myString << endl;
    cout << pat;
    
    if(myString.find(pat) != string::npos ){
        answer = 1;
    }
    
    return answer;
}

File: 프로그래머스/0/181878. 원하는 문자열 찾기/원하는 문자열 찾기.java
class Solution {
    public int solution(String myString, String pat) {
        int answer = 0;
        
        for(int i=0; i< myString.length() - pat.length()+1; i++){
            
            String str =myString.substring(i,pat.length() + i).toLowerCase();
            
            if(str.equals(pat.toLowerCase())){
                answer=1;
            }
            
            
        }
        
        return answer;
    }
}